wrappers/services_gen_test.go

```diff
@@ -1019,3 +1021,158 @@ OOMScoreAdjust=-899
 WantedBy=multi-user.target
 `, mountUnitPrefix, mountUnitPrefix))
 }
+
+func (s *servicesWrapperGenSuite) TestQuotaGroupSlice(c *C) {
+	service := &snap.AppInfo{
+		Snap: &snap.Info{
+			SuggestedName: "snap",
+			Version:       "0.3.4",
+			SideInfo:      snap.SideInfo{Revision: snap.R(44)},
+		},
+		Name:         "app",
+		Command:      "bin/foo start",
+		Daemon:       "simple",
+		DaemonScope:  snap.SystemDaemon,
+		RestartDelay: timeout.Timeout(20 * time.Second),
```

Comment:
(nitpick) I guess this is also used in the other tests but it seems this setting is irrelevant for what we test so we could also just not have it here.

Reply:
Nice catch, thank you! fixed